pixi.toml

```diff
@@ -47,24 +47,31 @@ ucal = { git = "https://github.com/nsls-ii-sst/ucal", rev = "v0.1.0" }
 bluesky-queueserver = "*"
 bluesky-httpserver = "*"
 
+[feature.gui.dependencies]
+pyqt = "*"
+qtconsole = "*"
+qt-wayland = ">=5.15.15,<6"
+
 [feature.qs.tasks]
 # This section needs some development work to understand the best way to
 # thread the required configuration through both the task and files checked into
 # the profile.  These will interact with the ansible roles for pushing out the
 # systemd units
-qs-backend = "start-re-manager --profile-dir=."
+qs-backend = "start-re-manager --redis-addr redis:6379 --zmq-publish-console ON --use-ipython-kernel ON --ipython-kernel-ip auto --startup-profile collection"
```

Comment:
you probably want to keep `--profilie-dir=.` so it uses the profile is this directory not the one in `.ipython/profile_collection`